pkg/service/audit_utils.go

```diff
@@ -386,46 +385,14 @@ func CreateApiKeyAuditEvent(ctx context.Context, al *zap.Logger, action string,
 			"apikey": id,
 		},
 	}
-	if err := audit.CreateV1Event(al, sd, detail, fmt.Sprintf("apikey.%s.success", action), ""); err != nil {
-		_log.Warn("unable to create audit event", err)
-	}
-}
-
-func GenerateApiKeyAuditEvent(ctx context.Context, al *zap.Logger, action string, id string) {
-	sd, ok := GetSessionDataFromContext(ctx)
-	if !ok {
-		_log.Warn("unable to create audit event: could not fetch info from context")
-		return
-	}
-
-	detail := &audit.EventDetail{
-		Message: fmt.Sprintf("ApiKey %sd for user %s", action, id),
-		Meta: map[string]string{
-			"apikey": id,
-		},
+	if action == AuditActionDownload {
```

Comment:
this is not required as we do not have any option for users to download just the apikey.